tests/gold_tests/pluginTest/slice/slice.test.py

```diff
@@ -87,19 +87,16 @@
     ' @plugin=slice.so @pparam=--blockbytes-test={}'.format(block_bytes)
 ])
 
-# minimal configuration
-ts.Disk.records_config.update({
-#  'proxy.config.diags.debug.enabled': 1,
-#  'proxy.config.diags.debug.tags': 'slice',
-  'proxy.config.http.cache.http': 1,
-  'proxy.config.http.wait_for_cache': 1,
-})
+# ts.Disk.records_config.update({
+#   'proxy.config.diags.debug.enabled': 1,
```

Comment:
Nitpick:  You could uncomment this call, but set debug.enabled to 0.  Then it would only be necessary to change one character to temporarily enable debug tracing.

Reply:
That's a good point.  I like that better.